doc/rtd/howto/run_cloud_init_locally.rst

```diff
@@ -70,6 +70,11 @@ Download an Ubuntu image to run:
 
     wget https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
 
+.. note::
+   Please be aware, the example is written for x86. It works on non-x86 but
+   is slower as it will need to be emulated. You can change the image type and
+   qemu-system variant to your architecture to make it faster for you.
```

Comment:
```suggestion
   qemu-system command name to your architecture to make it faster for you.
```

Reply:
I'll fix it slightly differently but yeah - good catch